pkg/state/factory.go

```diff
@@ -123,14 +123,19 @@ func newNicClusterPolicyStates(k8sAPIClient client.Client, scheme *runtime.Schem
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create ib-kubernetes State")
 	}
+	nvIpamCniState, err := NewStateNVIPAMCNI(
+		k8sAPIClient, scheme, filepath.Join(manifestBaseDir, "state-nv-ipam-cni"))
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to create ib-kubernetes State")
+	}
 
 	return []Group{
 		NewStateGroup([]State{podSecurityPolicyState}),
 		NewStateGroup([]State{multusState, cniPluginsState, ipoibState, whereaboutState}),
 		NewStateGroup([]State{ofedState}),
 		NewStateGroup([]State{sriovDpState}),
 		NewStateGroup([]State{sharedDpState, nvPeerMemState}),
-		NewStateGroup([]State{ibKubernetesState}),
+		NewStateGroup([]State{ibKubernetesState, nvIpamCniState}),
```

Comment:
@adrianchiris Why add the nvIpamCNI to this stategroup? Shouldn't it belong to the multus & cniPlugins stategroup?

Reply:
it doesnt really matter IMO, i have a refactor PR to remove the whole StateGroup stuff